lib/new_relic/agent/configuration/default_source.rb

```diff
@@ -38,6 +38,12 @@ def self.instrumentation_value_of(disable_key, prepend_key = nil)
         end
       end
 
+      def self.instrumentation_boolean_value_of(key)
```

Comment:
These are great points! I was struggling with the name and like your suggestion, `instrumentation_value_from_boolean` a lot. I was trying to make the `instrumentation_value_of` method work for this, but was hitting a bit of a wall since it seems like it sets the opposite value. Perhaps we could pair on it?

Reply:
Decided during mobbing to use `instrumentation_value_from_boolean` and leave the two methods separated.